src/components/solid/FaqAccordion.tsx

```diff
@@ -0,0 +1,55 @@
+import { SolidMarkdown } from 'solid-markdown';
+import {
+  Accordion,
+  AccordionContent,
+  AccordionItem,
+  AccordionTrigger,
+} from '~/components/ui/Accordion';
+
+const content = [
+  {
+    id: '1',
+    question: "My notifications aren't showing?",
+    answer:
+      "Some organisations require applications to request access before allowing access to any data (including notifications) about their repositories.\n\nTo check if Gitify is approved by your organisation you can go to [settings](https://github.com/settings/applications), then click on **Gitify** and scroll to **Organization access**.\n\nAlternatively, if you generated a **PAT**, you can check if it has the correct permissions by going to [settings](https://github.com/settings/tokens) and checking if the token's SSO has been Authorized.",
```

Comment:
Do we need to reword this FAQ? Perhaps all it needs is a more specific header? 

I've reread it a dozen times and it seems to be covering 
s
- GitHub App or GitHub Oauth App approval for certain orgs (enterprise??)
- PAT when used with an org that has SSO/ SAML configured?

Reply:
The SSO part was added in this PR 👀

Reply:
Yes I know... But I don't think regular GitHub Cloud users with PAT can configure that...

Reply:
Maybe to put it differently, if I'm a regular GitHub Cloud user, and "my notifications aren't showing" - these tips aren't applicable.

Reply:
Sounds good! Alright, I'll merge this in and open another one in a few hours!